src/builtins/unalias.c

```diff
@@ -0,0 +1,28 @@
+/*
+    psh/backends/posix2/run.c - process-related posix backend
+    Copyright 2020 Manuel Bertele
+
+    This file is part of Psh, P shell.
+
+    Psh is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Psh is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include "backend.h"
+#include "builtin.h"
+#include "libpsh/util.h"
+
+int builtin_unalias(int argc, char **argv, psh_state *state)
```

Comment:
I'll probably place `alias` and `unalias` into the same file, otherwise we might end up with too many files in the `builtins` files lol